src/main.rs

```diff
@@ -1,9 +1,12 @@
 mod day_one;
+mod day_three;
 mod day_two;
 
 fn main() {
     println!("Day One:");
     day_one::execute();
     println!("Day Two:");
     day_two::execute();
+    println!("Day Three:");
+    day_three::execute();
```

Comment:
Doing all problems sequentially is fine for now, but what happens when you have one problem that has an unoptimized solution that takes a long time to run (*cough* Day 15 Part 2), blocking future problems from running? Consider allowing a way to specify a specific problem/day to run.

Reply:
I kind of like the idea of having all the days problems show up one by one like some sorta startup checklist 😆.

Did want to make a minor change (https://github.com/boarnoah/advent_of_code_2021/pull/4) which will swap between inputs vs samples. In a future case where the later algorithms are pretty slow, this way it won't impede quick iteration.